pkgs/development/libraries/java/jffi/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchFromGitHub, jdk, jre, ant, libffi, texinfo, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "jffi";
+  version = "1.3.9";
+
+  src = fetchFromGitHub {
+    owner = "jnr";
+    repo = "jffi";
+    rev = "jffi-${version}";
+    sha256 = "sha256-VjZYhMbad+AesANG06umRzqMWj+Ebzu59TYK7Tm/bFo=";
+  };
+
+  nativeBuildInputs = [ jdk ant texinfo pkg-config ];
+  buildInputs = [ libffi ] ;
+
+  buildPhase = ''
+    # The pkg-config script in the build.xml doesn't work propery
+    # set the lib path manually to work around this.
+    export LIBFFI_LIBS="${libffi}/lib/libffi.so"
```

Comment:
Shouldn't we fix it rather than setting always the env?

Reply:
@SuperSandro2000 The error I get when not setting this is:

```
 last 10 log lines:
              >     [javac] warning: [options] bootstrap class path not set in conjunction with -source 8
              >     [javac] 1 warning
              >
              > -build-native-library:
              >      [exec] make: *** No rule to make target '-L/nix/store/20ix3np9v02ph8fwb2v41r5mzlfg8f73-libffi-3.4.2/lib/../lib64', needed by '/build/source/build/jni/libjffi-1.2.so'.  Stop.
              >
              > BUILD FAILED
              > /build/source/build.xml:372: exec returned: 2

```

So I'm not sure if the `libffi.pc` file is wrong or if `jffi` uses it in an incompatible way.